fili-core/src/main/java/com/yahoo/bard/webservice/logging/LogFormatter.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.logging;
+
+/**
+ * Interface for log formatter that are meant to facilitate log exploration and data extraction.
+ */
+public interface LogFormatter {
+
+    /**
+     * Takes in a log block and returns a formatted String to be logged.
+     *
+     * @param logBlock  the log block that represents a log and that will be formatted in to the needed format.
+     *
+     * @return a formatted String to be logged
+     */
+    String write(LogBlock logBlock);
```

Comment:
It's strange that this method is called `write` and not `format` when the interface is a Formatter. Can we change the name of the method to `format`? (it's also not supposed to be writing anything, only returning the LogBlock as a formatted string.)